### ISSUE TITLE
Ruby 3.0 support

### ISSUE THREAD

**Issue by denny** [applandinc/appmap-ruby#126]
Hi, is there any news on when Ruby 3.0 might be supported?

Currently when I try it, my tests fail very early (at what I think is the first method call during pre-test setup) with this error:
```
  ArgumentError:
    wrong number of arguments (given 1, expected 0; required keyword: category)
```
Googling that found me [this similar-sounding issue](https://github.com/funbox/sidekiq-influxdb/issues/12) in what I think is another one of your projects? No idea if there's any connection or not though 🙂

**Comment by kgilpin** [comment]
Hi, thanks for the nudge on this. Now that a little time has passed, it looks like Rails now runs on Ruby 3.0. So it's definitely worth it for us to revisit this issue. We will take a look and report back here.

**Comment by stixsmith** [comment]
I think this is fixed on Ruby 3.0.x by simply changing line 370 in lib/app/config.rb from

Config.new name, config_params

to

Config.new name, packages: config_params[:packages]

This change seems to also still work on Ruby 2.5.x too

**Comment by kgilpin** [comment]
Thanks for the tip, I will check this out. We are doing a new batch of work on this repo, and we are planning to solve the issues with 3.0.

**Comment by kgilpin** [comment]
In Ruby 3.0, a Hash will not automatically be converted to keyword arguments. But, the operator `**` can be used to do this explicitly. So, I'm trying that approach:

```
        Config.new name, **config_params
```

**Comment by kgilpin** [comment]
This is now available in 0.65.0 https://rubygems.org/gems/appmap/versions/0.65.0! Closing.

**Comment by LizzieMcG** [comment]
@stixsmith Thanks for your help! Would love to send you a free AppLand t-shirt to show our appreciation. Lmk how to get in touch if you're game!
